components/formats-gpl/src/loci/formats/in/LeicaSCNReader.java

```diff
@@ -230,7 +230,7 @@ protected void initTiffParser() {
     // newer files from Versa systems do not
     if (handler != null) {
       Image i = handler.imageMap.get(0);
-      tiffParser.setYCbCrCorrection(!"versa".equalsIgnoreCase(i.devModel));
+      tiffParser.setYCbCrCorrection(!("versa".equalsIgnoreCase(i.devModel) || "leica scn400".equalsIgnoreCase(i.devModel)));
```

Comment:
If the list of scanners keeps growing, should we start maintaining a static list at the reader level?